libs/langchain/langchain/llms/databricks.py

```diff
@@ -46,6 +46,10 @@ def post(
     ) -> Any:
         ...
 
+    @property
+    def llm(self) -> bool:
+        return False
```

Comment:
A flag to avoid passing completions params to a non-(external|foundational) endpoint.